src/context-selection/use-context-selection.js

```diff
@@ -0,0 +1,57 @@
+import { useCallback } from 'react'
+import {
+    ArrayParam,
+    StringParam,
+    useQueryParams,
+    useQueryParam,
+    withDefault,
+} from 'use-query-params'
+
+const PARAMS_SCHEMA = {
+    dataSetId: StringParam,
+    orgUnitId: StringParam,
+    periodId: StringParam,
+    categoryOptionComboSelection: withDefault(ArrayParam, []),
+    sectionFilter: StringParam,
+}
+
+export function useDataSetId() {
+    return useQueryParam('dataSetId', PARAMS_SCHEMA.dataSetId)
+}
+
+export function useOrgUnitId() {
+    return useQueryParam('orgUnitId', PARAMS_SCHEMA.orgUnitId)
+}
+
+export function usePeriodId() {
+    return useQueryParam('periodId', PARAMS_SCHEMA.periodId)
+}
+
+export function useCategoryOptionComboSelection() {
```

Comment:
I think we should call this `attributeOptionCombo`. This is the term we are using when talking about the `categoryOptionCombo` when it's on the dataSet. It's also the property-name in the API when using this value to get/set dataValues. 
It also prevents confusion with `categoryOptionCombos` on dataElements.

Reply:
Done